tests/ssid_set_snmp.py

```diff
@@ -24,17 +24,37 @@ def runTest(self):
         for wifi_device in wifi_name:
             if wifi_device == 'wifi_2g':
                 self.index = '.32'
-                self.set_ssid_name = 'SSIDwifi_2G'
+                set_ssid_name = 'SSIDwifi_2G'
             elif wifi_device == 'wifi_5g':
                 self.index = '.92'
-                self.set_ssid_name = 'SSIDwifi_5G'
-
+                set_ssid_name = 'SSIDwifi_5G'
             """Set SSID via SNMP"""
-            ssid_name = self.config.board['station']+self.set_ssid_name
+            ssid_name = config.board['station']+set_ssid_name
+            wifi_pass = config.wifi_password[0]
```

Comment:
OK.. I think you need to squash these commits down into fewer commits. It's not readable... now I see here you did make the SSID more random.

I don't think we should put the wifi password in the config, we can just use a random one as well.